public/app/features/explore/TraceView/createSpanLink.test.ts

```diff
@@ -215,14 +215,71 @@ describe('createSpanLinkFactory', () => {
       expect(linkDef).toBeUndefined();
     });
   });
+
+  describe('should return splunk link', () => {
+    const splunkUID = 'PD90232BAD06BE469';
```

Comment:
Let's use the string 'splunkUid' for this value since the current hard-coded value is specific to the instance of the Splunk datasource we're using for development